app/src/main/java/io/appium/uiautomator2/model/settings/MjpegBilinearFiltering.java

```diff
@@ -40,6 +42,9 @@ public Boolean getValue() {
 
     @Override
     protected void apply(Boolean value) {
+        if (value == null) {
+            throw new UiAutomator2Exception(String.format("Invalid %s value specified", SETTING_NAME));
```

Comment:
There is a dedicated exception for Illegal arguments

Reply:
same below

Reply:
OK. I'll update them all.

Reply:
This will be resolved w/ the upcoming update(s)